scripts/src/server.ts

```diff
@@ -316,18 +334,34 @@ class _Server implements Server {
 				cors: false | Cors.Options,
 				logger: winston.LoggerInstance,
 				files: Map<string, string>,
+				folders:Array<string>,
+				sessionConfig: Partial<SessionConfig>|undefined,
 				routes: Route[]) {
 		this.host = host;
 		this.port = port;
 		this.logger = logger;
 
 		this.koa = new Koa();
+
+		if (folders.length) {
+			folders.forEach((folder)=>{
```

Comment:
(formatting)
add spaces, also no need to wrap one param in brackets:
`folders.forEach(folder => {`